tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -3326,6 +3237,15 @@ Status ConvertStridedSlice(OpConverterParams* params) {
         "Length of begin, end, and stride must be equal, at ", node_def.name());
   }
 
+  // The slice op has many ways to define the actual operation that needs to be
+  // performed. We use ValidateStridedSliceOp to map the input parameters to
+  // begin, end, & strides. ValidateStridedSliceOp makes an effort to set known
+  // (static) begin/end/strides parameters. On return, begin, end, stride,
+  // processing_shape has the same rank as input. final_shape has extra dims
+  // added/removed. Negative indices are converted to positive values if the
+  // array size is known (which is the case in implicit and explicit batch
+  // mode). In dynamic shape mode final_shape and processing_shape can have
+  // unknown dimensions, even if [begin, end] defines a known shape.
```

Comment:
Can we revise the ending part "Negative indices are converted to positive values if the
  // array size is known (which is the case in implicit and explicit batch
  // mode). In dynamic shape mode final_shape and processing_shape can have
  // unknown dimensions, even if [begin, end] defines a known shape."
In particular, can we remove the mentioning of implicit batch mode, explicit batch mode and dynamic batch mode. I think what really matter is not the mode, but whether there is unknown dimensions in the shape. So may be this is good:
Negative values in begin/end/stride are converted to positive values to produce a known processing_shape if the input shape is static. Otherwise, processing_shape may contain unknown dimension values.